docs/resources/docker.md.erb

```diff
@@ -136,7 +136,7 @@ Or execute the profile directly via URL:
       its('commands') { should_not include '/bin/sh' }
       its('images') { should_not include 'u12:latest' }
       its('ports') { should include '0.0.0.0:1234->1234/tcp' }
-      its('labels') { should include 'License=GPLv2,Vendor=CentOS' }
+      its('labels') { should include ['License=GPLv2', 'Vendor=CentOS'] }
```

Comment:
This is probably not what is intended.  This would indicate that you are expecting `labels` to be and Array containing a nested Array containing two Strings in exactly that order.

The existing code was more accurate but not very usable - the `labels` field would be a String, and you can search a string for substrings using `include`.

Calling `include` on an Array does something completely different, looking at each element in isolation - it does not look for "subarrays", as your example would imply.

If you want to look for two different labels when `labels` is an Array, you'll want to do so in two tests:

```
# labels is an Array of Strings; look for individual elements
its('labels') { should include 'License=GPLv2' }
its('labels') { should include 'Vendor=CentOS' }
```

Reply:
I think while not ideal, the way I have it is correct. `docker.containers.labels` returns and Array of Arrays. Which is why I highlighted in the first comment on the PR my hesitancy (https://github.com/inspec/inspec/pull/3673#issuecomment-448057106)

Unless I'm misunderstanding something?:

```
   11: it 'check docker container parsing' do
    12:   require 'pry'; binding.pry
 => 13:   _(resource.containers.ids).must_equal ['3def9aa450f8bd772c3d5b07e27ec934e5f58575e955367a0aca2d93e0687536', 'd94f854370d2b02912e8fc636502bc72b74fbd567a7eba3fc6a52045bb28904e', '5a83c301f30ccd48579a74a84af6fdd0c0e0d66aacc7bb52abfa2ba2544c6c0c', '5a83c301f30ccd48579a74a84af6fdd0c0e0d66aacc7bb52abfa2ba2544c6c0c']
    14:   _(resource.containers.names).must_equal ['sleepy_khorana', 'laughing_austin', 'heuristic_almeida', 'laughing_lamport']            
    15:   _(resource.containers.labels).must_equal [['app=example', 'version=1.5.4'], [], [], []]                                           
    16: end

[1] pry(#<Inspec::Resources::Docker::docker>)> resource.containers.labels                                                                   
=> [["app=example", "version=1.5.4"], [], [], []]
[2] pry(#<Inspec::Resources::Docker::docker>)> resource.containers.labels.include?('app=example')                                           
=> false
[3] pry(#<Inspec::Resources::Docker::docker>)> 
```

Reply:
I think OR operations only are fine here! I'll update and change the examples.